### PR TITLE
Improves dice tactical combat for deaf

### PR DESCRIPTION
**Pull request by optimumtact**
:cl: oranges
tweak: You can now hear dice
/:cl:

merge token #18556

### CODE REVIEW

**Comment by TheColdTurtle** [comment]
the fuck does this mean

File: code/game/objects/items/weapons/dice.dm [hunk 1]
```diff
@@ -115,6 +115,7 @@
 
 /obj/item/weapon/dice/proc/diceroll(mob/user)
 	result = rand(1, sides)
+	fake_result = rand(1, sides)
```
Comment (RemieRichards) [inline code/game/objects/items/weapons/dice.dm]: this var doesn't exist

**Comment by coiax** [comment]
It means that blind people hear a random number rolled, which may or may not be the same as the actual number.

**Comment by Supermichael777** [comment]
This is actually kind of funny.  add one for flipping coins.

**Pull request merged**